src/Response/BadRequestApiResponse.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Response;
+
+use Symfony\Component\HttpFoundation\Response;
+
+class BadRequestApiResponse extends ApiResponse
+{
+    public function __construct(mixed $errors)
+    {
+        parent::__construct(null, $errors, Response::HTTP_BAD_REQUEST);
```

Comment:
jak się zastosujesz do tego co wyżej pisałem to wtedy:
```suggestion
        parent::__construct(
            errors: $errors, 
            httpStatusCode: Response::HTTP_BAD_REQUEST
        );
```
Nie musisz podawać wtedy pierwszego parameteru bo jest domyślnie null